week2/jalalhm/test2.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+
+const fs = require('fs');
+
+const TODOS_FILENAME = process.env["TODOS_FILENAME"] || "todos.json";
+
+let myTodoList = []
+
+// Helper function definitions
+
+let readTodosFile = (filename) => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filename, "utf8", (err, data) => {
+            if (err) {
+                throw err
+                reject()
```

Comment:
It might make more sense here to `reject()` with the error and let the caller handle it, rather than throwing the error and then rejecting the promise.